docs/usage/user-stories/swissquote.md

```diff
@@ -0,0 +1,229 @@
+
+### A word on Renovate
+
+> The next section could look like an ad or a sponsored post; it’s not. We’re just big fans of the product.
+
+The ease of getting started with Renovate’s Docker image is what got us onboard easily.
+But what confirmed our choice is the insane crazy amount of features and [configuration options](https://docs.renovatebot.com/configuration-options/).
+
+Some features and options we enjoy:
+
+- [Shared configurations (presets)](https://docs.renovatebot.com/key-concepts/presets/), we have a Swissquote default configuration we set for all repositories, each team can extend it with their own practices
+- [Integration with GitHub’s Dependabot alerts](https://docs.renovatebot.com/configuration-options/#vulnerabilityalerts) to raise the priority and send security remediation PRs as soon as possible
+- Each rule can be customized either globally [or specified per package](https://docs.renovatebot.com/configuration-options/#packagerules)
+- Works with your [private package registry](https://docs.renovatebot.com/getting-started/private-packages/)
+- Supports dozens of [languages and package managers](https://docs.renovatebot.com/modules/manager/#supported-managers): Maven, Docker, NPM, Docker Compose, Python
+- If you are using dependencies in a custom way, [there is a special regexManager](https://docs.renovatebot.com/configuration-options/#regexmanagers) that allows you to transform patterns into dependencies
+
+There is an [on-premise option](https://www.mend.io/free-developer-tools/renovate/on-premises/), but you can also have it as [an app on GitHub.com](https://github.com/marketplace/renovate).
+On our side, we’re not using the on-premise but rather a custom scheduler using the open source Docker image.
+
+## Some stats after two years with Renovate
+
+We started using Renovate Bot in 2019, using the (now deprecated) `renovate/pro` Docker image.
+We installed it as a GitHub app and some early adopters started to use it.
+
+Pretty quickly, we ran into the biggest limitation; this Docker image runs all repositories one after another.
+A single loop was taking hours and made it very difficult to check the logs as they weren’t separated per repository.
+
+This is why we created our own scheduler; each hour, all repositories would be queued to run, and GitHub app events would schedule a single repository.
+We started to collect metrics and store the logs separately for each repository.
+
+Here is the dashboard for our current scheduler:
+
+![A dashboard we made at Swissquote to keep our Renovate runs in check, July 2022](./swissquote_stats.png)
+
+We don’t force any team to use Renovate, each team can decide to opt-in and do it for each project separately.
+
+- 824 repositories enabled out of about 2000 active repositories
+- 8000 PRs were merged since we installed Renovate
+- 239 PRs were merged last month
+- 2 SSDs died on our Renovate machine with the number of projects to clone again and again
```

Comment:
```suggestion
Some statistics:

- 824 repositories enabled out of about 2000 active repositories
- 8000 PRs were merged since we installed Renovate
- 239 PRs were merged last month
- 2 SSDs died on our Renovate machine with the number of projects to clone again and again
```

I feel we need a "platform sentence" for the list.